plugin.cpp

```diff
@@ -55,11 +55,19 @@ static const char *default_config = QUOTE({
 		"displayName": "Send JSON",
 		"options" : ["Objects","Strings"]
 		},
+	"compression": {
+		"description": "Type of compression codec to be used before sending data to kafka broker",
```

Comment:
"The compression codec to be used to send data to Kafka broker"

Reply:
updated